lib/spack/docs/tutorial_buildsystems.rst

```diff
@@ -475,20 +488,16 @@ In the :code:`CMakePackage` class we can override the following phases:
 The :code:`CMakePackage` class also provides sensible defaults so we only need to
 override :code:`cmake_args()`.
 
-Let's look at these defaults in the :code:`CMakePackage` class:
+Let's look at these defaults in the :code:`CMakePackage` class in the :code:`_std_args()` method:
 
 .. code-block:: console
 
     $ spack edit --build-system cmake
 
-
-And go into a bit of detail on the highlighted sections:
-
-
 .. literalinclude:: ../../../lib/spack/spack/build_systems/cmake.py
    :language: python
-   :lines: 18-73, 75-136, 155-192
-   :emphasize-lines: 38,49,67,75,77,80,81,82,83,92,98,116,117
+   :lines: 102-147
+   :emphasize-lines: 10,18,24,36,37,38,44
```

Comment:
@mamelara this is emphasizing some lines that aren't included by the previous line, does that work?

Reply:
Yes, it does actually. It's based off of the total length of the lines I am using from the original file.